recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
Yep, my conda installation also appears to have `tclsh8.5` and not `tclsh`, so I think we should either go back to using `tclsh8.5` here, or adjust the conda packages to also install it as `tclsh`.